### PR TITLE
[PATCH qa] QA Issues 

### PR DESCRIPTION
**Pull request by damassi**
- Make JSON serialization safer (**TODO**: figure out what's causing circular ref issues *only* on staging)
- Pass in initial breakpoint for responsive SSR rendering
- Fix Current event square
- Fix Overview grid filter layout

### CODE REVIEW

File: src/Styleguide/Pages/Boot.tsx [hunk 1]
```diff
@@ -12,10 +13,19 @@ injectGlobalCSS(`
   }
 `)
 
-export const Boot = ({ children }) => {
+export interface BootProps {
+  initialState?: {
+    breakpoint: Breakpoint
+  }
```
Comment (damassi) [inline src/Styleguide/Pages/Boot.tsx]: This was the best i could come up with right now in the 11th hour :/

File: src/Router/AppShell.tsx [hunk 2]
```diff
@@ -4,7 +4,15 @@ import { ContextProvider } from "../Components/Artsy"
 import { AppShellProps } from "./types"
 
 export const AppShell: SFC<AppShellProps> = props => {
-  const { loadableState, data, children, provide } = props
+  const { loadableState, data = {}, children, provide } = props
+
+  let hydrationData
+  try {
+    hydrationData = serialize(data, { isJSON: true })
```
Comment (zephraph) [inline src/Router/AppShell.tsx]: 👍

Comment (damassi) [reply src/Router/AppShell.tsx]: this file is so ugly... really need to refactor all of this

Comment (zephraph) [reply src/Router/AppShell.tsx]: We'll get there.

**Pull request merged**